net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -272,48 +275,49 @@ mwan3_set_connected_ipv4(){
 
 mwan3_set_connected_iptables()
 {
-	local connected_network_v6 source_network_v6
+	local connected_network_v6
 
 	$IPS -! create mwan3_connected_v4 hash:net
 	$IPS create mwan3_connected_v4_temp hash:net
 
 	mwan3_set_connected_ipv4
 
-	$IPS -! create mwan3_connected_v6 hash:net family inet6
-	$IPS create mwan3_connected_v6_temp hash:net family inet6
+	[ $NO_IPV6 -eq 0 ] && {
```

Comment:
Please quote all `$NO_IPV6` checks.

Reply:
NO_IPV6 is set via the output of a previous command, so will be an integer, and shellcheck is able to detect that and not warn about it.